docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -71,15 +19,32 @@ or
 gst-launch-1.0 filesrc location=/path/to/h264/file ! decodebin ! videoconvert ! ximagesink sync=false
 ```
 
-If you're using tcp, you can connect the stream directly with your phones ip address using
+If you're using tcp, you can connect the stream directly with your phone's IP address using
 ```
 gst-launch-1.0 tcpclientsrc host=<Device IP Address> port=3000 ! decodebin ! videoconvert ! ximagesink sync=false
 ```
 
+### HMI Modifications
+Using either pipe or socket streaming may require a modification to the SDL HMI.
+
+#### VIDEO
+Comment out the following line in `app/model/sdl/Abstract/Model.js`:
+```
+//  SDL.SDLModel.playVideo(appID);
+```
+
+#### AUDIO
+Comment out the following lines in `app/model/sdl/Abstract/Model.js`:
+```
+//  SDL.StreamAudio.play(
+//      SDL.SDLController.getApplicationModel(appID).navigationAudioStream
+//  );
+```
+
 ## Pipe Streaming
 
 ### Configuration (smartDeviceLink.ini)
-In the build folder directory, open bin/smartDeviceLink.ini in a text editor and the make the following changes:
+In the Core build folder, open bin/smartDeviceLink.ini and the make the following changes:
```

Comment:
```suggestion
In the Core build folder, open `bin/smartDeviceLink.ini` and the make the following changes:
```

Reply:
Also, the Configuration section for socket streaming should have the same description for modifying the ini file, just to keep it consistent.

Reply:
good idea, fixed in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/b07a1990e1799725a09996753b1db4bba73b664b